contracts/erc721/UserMintableERC721.sol

```diff
@@ -68,18 +64,22 @@ contract UserMintableERC721 is Initializable, ERC721, ERC721Enumerable, ERC721Me
   bytes constant internal ROLES_NFTS = hex"00000004";
   // compact prop for "signatures_tree.signatures"
   bytes constant internal SIGNATURE_TREE_SIGNATURES = hex"0300000000000001";
-  // compact prop for "signature" for a signature tree signature
-  bytes constant internal SIGNATURE_TREE_SIGNATURES_SIGNATURE = hex"00000004";
 
   // Constants used as values
   // Value for a Read Action. 1 means is has Read Access
   bytes constant internal READ_ACTION_VALUE = hex"0000000000000002";
   // Value for invoice status. bytes for 'unpaid'
-  bytes constant internal INVOICE_STATUS_UNPAID = hex"756e70616964";
+  bytes constant internal INVOICE_STATUS_UNPAID = hex"756e706169640000000000000000000000000000000000000000000000000000";
```

Comment:
As we are using now fixed length for some fields and status is one of them, with fixed length the value is filled with 0s. This causes the max line constraint to fail

Reply:
Added disable line as @rdinicut suggested.